sources/packages/backend/libs/integrations/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -64,10 +68,16 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         application: { student: { sinValidation: true, user: true } },
         offering: { institutionLocation: true, educationProgram: true },
       },
-      where: {
-        assessmentDate: dateEqualTo(processingDate),
-        offering: { institutionLocation: { hasIntegration: true } },
-      },
+      where: [
+        {
+          assessmentDate: dateEqualTo(processingDate),
```

Comment:
When we put an array of conditions in where, what does it mean OR ?

Reply:
yes, its an OR condition